dartagnan/src/test/java/com/dat3m/dartagnan/miscellaneous/AnalysisTest.java

```diff
@@ -73,8 +73,8 @@ public void dependencyMustOverride() throws InvalidConfigurationException {
         b.addChild(0, e5);
 
         Program program = b.build();
-        LoopUnrolling.newInstance().run(program);
         Compilation.newInstance().run(program);
+        LoopUnrolling.newInstance().run(program);
```

Comment:
This reordering will be unnecessary if you remove the preconditions again (also holds for all the other tests).

Reply:
I will keep them simply to be consistent to the order the `ProcessorManager` applies transformations